### PR TITLE
Feat/262 meal planner style

### LINKED ISSUE #262: feat(meal-planner): style of meal planner (MEDIUM)

### PR DESCRIPTION
**Pull request by FangchengDou**
close #262

### CODE REVIEW

File: source/components/meal-planner/meal-planner-recipe-card/meal-planner-recipe-card.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { YummyRecipesComponent } from "/components/core/yummy-recipes-component.js";
+
+class MealPlannerRecipeCard extends YummyRecipesComponent {
+  constructor() {
+    super();
+    this.htmlPath =
+      "components/meal-planner-recipe-card/meal-planner-recipe-card.html";
```
Comment (jbutera94) [inline source/components/meal-planner/meal-planner-recipe-card/meal-planner-recipe-card.js:7]: This should be `"components/meal-planner/meal-planner-recipe-card/meal-planner-recipe-card.html"`

**Review by jbutera94 — APPROVED** [review]
Looks good! Noticed one small typo

**Comment by FangchengDou** [comment]
I don't understand the codacy issue. If you don't mind just merge it.

**Comment by jbutera94** [comment]
I think it means it wants the CSS selectors in a specific order. I can fix it and then merge it

**Pull request merged**